File: frontend/src/app/models/vet-service.ts
export interface VetService {
  id?: number;
  name: string;
  description: string;
  duration: number;  // in minutes
  price: number;
  icon: string;
  iconBg: string;
}

export interface NewVetService extends Omit<VetService, 'id'> {} 

File: backend/src/main/java/com/vet/system/model/VetService.java
package com.vet.system.model;

import jakarta.persistence.*;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

@Entity
@Table(name = "services")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class VetService {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    
    private String name;
    private String description;
    private Integer duration;  // in minutes
    private Double price;
    private String icon;
    private String iconBg;
} 